packages/compass-aggregations/src/components/pipeline-builder-input-documents.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useMemo } from 'react';
+import { connect } from 'react-redux';
+import type { Document as DocumentType } from 'mongodb';
+
+import { KeylineCard, css, cx, spacing, palette, useDarkMode, Body, IconButton, Icon } from "@mongodb-js/compass-components";
+import { Document } from '@mongodb-js/compass-crud';
+
+import { refreshInputDocuments } from '../modules/input-documents';
+import LoadingOverlay from './loading-overlay';
+
+const cardStyles = css({
+  margin: `0 ${spacing[3]}px`,
+  marginBottom: spacing[3]
+});
+
+const headerStyles = css({
+  display: 'flex',
+  gap: spacing[1],
+  alignItems: 'center',
+  width: '100%',
+  padding: spacing[2],
+});
+
+
+const headerTextStyles = css({
+  flex: 1,
+});
+
+const bodyStyles = css({
+  position: 'relative',
+  padding: spacing[3],
+  paddingBottom: spacing[2],
+  borderTopWidth: '1px',
+  borderTopStyle: 'solid'
+});
+
+const bodyStylesLight = css({
+  borderTopColor: palette.gray.light2
+});
+
+const bodyStylesDark = css({
+  borderTopColor: palette.gray.dark2
+});
+
+const documentsContainerStyles = css({
+  marginTop: spacing[3],
+  display: 'flex',
+  overflowX: 'scroll',
+  gap: spacing[3],
+  paddingBottom: spacing[2]
+});
+
+const documentContainerStyles = css({
+  flexShrink: 0,
+  width: '384px',
+  height: '150px',
```

Comment:
Looking at the designs it looks like there's a bit less padding than what we have here. Less padding gives more space to content so I think it's something we want to do. Also how do you feel about increasing the document preview size by 20 pixels?
| before | after |
| - | - |
| ![Screen Shot 2022-12-05 at 1 40 19 PM](https://user-images.githubusercontent.com/1791149/205717887-c5a81b63-5023-469e-aaea-0eadfd6ff2e2.png) | ![Screen Shot 2022-12-05 at 1 45 17 PM](https://user-images.githubusercontent.com/1791149/205717895-dc6b6dcb-7b4c-4175-9e8f-e8402abbc090.png) |

```suggestion
  padding: `${spacing[1]}px ${spacing[2]}px`,
});

const headerTextStyles = css({
  flex: 1,
});

const bodyStyles = css({
  position: 'relative',
  padding: `${spacing[2]}px ${spacing[2] + spacing[1]}px`,
  borderTopWidth: '1px',
  borderTopStyle: 'solid'
});

const bodyStylesLight = css({
  borderTopColor: palette.gray.light2
});

const bodyStylesDark = css({
  borderTopColor: palette.gray.dark2
});

const documentsContainerStyles = css({
  marginTop: spacing[2],
  display: 'flex',
  overflowX: 'scroll',
  gap: spacing[2],
  paddingBottom: spacing[2]
});

const documentContainerStyles = css({
  flexShrink: 0,
  width: '384px',
  height: '170px',
```